src/js/views/patients/sidebar/action/action-sidebar.hbs

```diff
@@ -35,4 +35,5 @@
   </div>
   {{/unless}}
 </div>
+<div data-comment-region></div>
```

Comment:
in the comp it looks to me like this is bottom aligned if there are no activities

Reply:
The activities container grows to fill the height regardless of how many activity events there are. So this stays bottom-aligned even if there is only the "created" event listed. At least in Chrome, I'll check other browsers.